runtime/cudaq/platform/default/rest_server/helpers/RestRemoteClient.cpp

```diff
@@ -236,9 +236,10 @@ class NvcfRuntimeClient : public RemoteRestRuntimeClient {
             m_functionId = funcId;
             if (m_logLevel > LogLevel::None) {
               // Print out the configuration
-              cudaq::log("Submitting jobs to NVQC service with {} GPU(s). Max "
-                         "wall time: {} seconds.",
-                         info.numGpus, info.timeoutSecs);
+              cudaq::log(
```

Comment:
FYI, this change is not duplicated in PyRestRemoteClient.cpp because this change is just bringing RestRemoteClient.cpp in alignment with what PyRestRemoteClient.cpp already had.